io.github.mki1967.mki3dgame.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id": "io.github.mki1967.mki3dgame",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.golang"
+    ],
+    "command": "mki3dgame",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=pulseaudio", "--share=network", "--device=dri", "--socket=wayland"],
+    "cleanup-commands": [
+	"rm -rf /app/go # remove working space"
+    ],
+    "modules": [
+	{
+	    "name": "io.github.mki1967.mki3dgame",
+	    "only-arches": [
```

Comment:
This can be removed since the `flathub.json` file prevents it building anywhere else.